dspace-server-webapp/src/main/java/org/dspace/app/rest/repository/CollectionRestRepository.java

```diff
@@ -169,8 +167,19 @@ public CollectionResource wrapResource(CollectionRest collection, String... rels
     }
 
     @Override
-    @PreAuthorize("hasAuthority('ADMIN')")
     protected CollectionRest createAndReturn(Context context) throws AuthorizeException {
+        throw new DSpaceBadRequestException("Missing parent parameter.");
+    }
+
+    @Override
+    @PreAuthorize("hasPermission(#id, 'COMMUNITY', 'ADD')")
+    protected CollectionRest createAndReturn(Context context, UUID id) throws AuthorizeException {
+
+        if (id == null) {
+            throw new DSpaceBadRequestException("The given parent parameter was invalid: "
+                + id);
```

Comment:
If the `id` is null here, we should return a message that says it's null (i.e. there's no point in printing out the value (as it'll always be null).  So, maybe reword this error to be similar to the one above (`"Parent Community UUID is null. Cannot create a Collection without providing a parent Community"`)